src/controllers/itemDetails/index.js

```diff
@@ -167,7 +167,30 @@ function renderTrackSelections(page, instance, item, forceReload) {
         return;
     }
 
-    const mediaSources = item.MediaSources;
+    var mediaSources = item.MediaSources;
+
+    var resolutionNames = [];
+    var sourceNames = [];
+    mediaSources.forEach(function (v) {
+        (v.Name.endsWith('p') || v.Name.endsWith('i')) ? resolutionNames.push(v) : sourceNames.push(v);
+    });
+
+    resolutionNames.sort((a, b) => parseInt(b.Name) - parseInt(a.Name));
```

Comment:
You should always specify a radix when calling `parseInt` as the behaviour can be unpredictable without it.